src/plugins/bfetch/public/batching/create_streaming_batched_function.test.ts

```diff
@@ -423,6 +445,44 @@ describe('createStreamingBatchedFunction()', () => {
       expect(result3).toEqual({ b: '3' });
     });
 
+    describe('when requests are aborted', () => {
```

Comment:
Could also please add a test that checks that the HTTP request is aborted when all items in the batch are aborted. I.e. the `signal` passed to `fetchStreamingInjected` is aborted.